wicket-tool/src/java/org/sakaiproject/wicket/model/SimpleDateFormatPropertyModel.java

```diff
@@ -18,23 +18,26 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
-public class SimpleDateFormatPropertyModel extends DecoratedPropertyModel {
-
+public class SimpleDateFormatPropertyModel extends DecoratedPropertyModel
+{
 	private static final long serialVersionUID = 1L;
 
 	private SimpleDateFormat dateFormat;
-	
-	public SimpleDateFormatPropertyModel(Object modelObject, String expression) {
+
+	public SimpleDateFormatPropertyModel(Object modelObject, String expression)
+	{
 		super(modelObject, expression);
 		this.dateFormat = new SimpleDateFormat("MM/dd/yyyy hh:mm a");
 	}
 
 	@Override
-	public Object convertObject(Object object) {
-		if (object instanceof Date) {
+	public Object convertObject(Object object)
+	{
+		if (object instanceof Date)
```

Comment:
Use generics to avoid this check (ie. SimpleDateFormatPropertyModel extends DecoratedPropertyModel<Date>. Could also upgrade this to use Java 8 date classes.